explainer.md

```diff
@@ -0,0 +1,802 @@
+
+The `.minute` property represents the minute of the hour of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.second: number
+
+The `.second` property represents the seconds of the minute of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.millisecond: number
+
+The `.millisecond` property represents the milliseconds of the seconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.microsecond: number
+
+The `.microsecond` property represents the microseconds of the milliseconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.nanosecond: number
+
+The `.nanosecond` property represents the nanoseconds of the microseconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.dayOfWeek: number
+
+The `.dayOfWeek` property represents the day of the week according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day where Monday is `1` and `Sunday` is `7` in accordance with ISO-8601. 
+
+#### OffsetDateTime.prototype.dayOfYear: number
+
+The `.dayOfYear` property represents the ordinal day of the Gregorian year according to ISO-8601.
+
+#### OffsetDateTime.prototype.weekOfYear: number
+
+The `.weekOfYear` property represents the ISO week-number. Beware that dates at the begining of a year may be part of a week from the preceding year, and dates at the end of a year may be part of a week at the beginning of the next year, as the first week of any year is defined as the week that contains the first Thursday of the week.
+
+#### OffsetDateTime.prototype.plus(data: IntervalLike): ZonedDateTime
+
+Creates a new `ZonedDateTime` object by adding (subtracting for negative values) values to its members. The specified values must be numeric if specified.
+
+The algorithm is such that:
+
+1. the individual values are added to the existing values.
+2. the range of `nanoseconds` is ensured to be between 0 and 999 by adjusting the `microseconds`
+3. the range of `microseconds` is ensured to be between 0 and 999 by adjusting the `milliseconds`
+4. the range of `milliseconds` is ensured to be between 0 and 999 by adjusting the `seconds`
+5. the range of `seconds` is ensured to be between 0 and 59 by adjusting the `minutes`
+6. the range of `minutes` is ensured to be between 0 and 59 by adjusting the `hours`
+7. the range of `hours` is ensured to be between 0 and 23 by adjusting `days`
+8. the range of `days` is ensured to be between 1 and 29-31 depending on the month by adjusting `month`
+9. the range of `months` is ensured to be between 1 and 12 by adjusting the `years`.
```

Comment:
The `plus` algorithm descriptions need rewording to reflect the differential naming of properties between ~~Interval~~ Duration vs. other types (plural in the former, singular in the latter).